lua/nvim-drawer/init.lua

```diff
@@ -276,6 +280,120 @@ function mod.create_drawer(opts)
     instance.state.previous_bufnr = final_bufnr
   end
 
+  function instance.get_win_config()
```

Comment:
This function + `nvim_win_set_config` kind of makes `set_size` moot, it does the same thing.